app/src/main/java/org/simple/clinic/appconfig/Country.kt

```diff
@@ -1,10 +1,21 @@
 package org.simple.clinic.appconfig
 
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
 import java.net.URI
 
+@JsonClass(generateAdapter = true)
 data class Country(
+
+    @Json(name = "country_code")
     val code: String,
```

Comment:
We should rename `code` to `countryCode` or `isoCode`.  We may end up adding more codes in the future so it might be ideal to specify what code is this one since all the codes are of type `String`.

Reply:
Makes sense. My long term plan is to move these types to [Inline classes](https://kotlinlang.org/docs/reference/inline-classes.html) when they are stable, so I guess naming them differently would help.